### PR TITLE
Fix/overview table

### PR DESCRIPTION
**Pull request by MauritsioRK**
Fixes the issue described here: https://alice.its.cern.ch/jira/browse/O2B-251
As well as some other small bug fixes.

### CODE REVIEW

**Comment by john-ghatas** [comment]
The tests are not functioning correctly right now we'll investigate tomorrow morning/evening :)

**Comment by mvegter** [comment]
@MauritsioRK , @john-ghatas What is the reason to leave out the column sizing and visiblity options from InfoLogger?

File: lib/public/app.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.cell-xs { width: 2rem; }
+.cell-s { width: 4rem; }
+.cell-m { width: 8rem; }
+.cell-l { width: 16rem; }
+.cell-xl { width: 32rem; }
+
+/* last column fills space */
+.cell-xs:last-child, .cell-s:last-child, .cell-m:last-child, .cell-l:last-child, .cell-xl:last-child { width: initial; }
+.cell-xs:last-child .resizeWidth, .cell-s:last-child .resizeWidth, .cell-m:last-child .resizeWidth, .cell-l:last-child .resizeWidth, .cell-xl:last-child .resizeWidth { visibility: hidden; }
```
Comment (mvegter) [inline lib/public/app.css:9]: Add an empty line at the end of the file.

File: lib/public/views/Logs/ColumnNames/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        name: 'Origin',
+        visible: true,
+        size: 'cell-m',
+    },
+    subtype: {
+        name: 'Subtype',
+        visible: true,
+        size: 'cell-m',
+    },
+
```
Comment (mvegter) [inline lib/public/views/Logs/ColumnNames/index.js:47]: Remove the unnecessary whiteline

File: test/public/index.js [hunk 3]
```diff
@@ -11,8 +11,8 @@
  * or submit itself to any jurisdiction.
  */
 
-const LogsSuite = require('./logs.test');
+const LogsSuite = require('./logs');
 
-module.exports = () => {
+module.exports = function () {
```
Comment (mvegter) [inline test/public/index.js:16]: Any reason not to use an arrow function here?

File: lib/public/views/Logs/ColumnNames/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const COLUMN_NAMES = {
```
Comment (mvegter) [inline lib/public/views/Logs/ColumnNames/index.js:14]: Not sure if `COLUMN_NAMES` fits here as it is responsible for a lot more.

File: test/public/index.js (hunk 3 above)
Comment (john-ghatas) [reply test/public/index.js:16]: Not really, we were trying to pass arguments to the tests but to no avail. Will turn it back to an arrow function.

File: lib/public/views/Logs/ColumnNames/index.js (hunk 4 above)
Comment (john-ghatas) [reply lib/public/views/Logs/ColumnNames/index.js:14]: Renaming it to active columns along with the folder where the file is in.

**Comment by john-ghatas** [comment]
Just checked the master branch (rebased a few hours ago already) and everything it up to date. Branch should be good to go now

**Pull request merged**